### DISCUSSION TITLE
I want login my account on other device

### DISCUSSION THREAD

**Discussion by alidehkhodaei** [community/community#45130] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, I can login only one device and when i want login another device, it say "incorrect username or password" but username and password are correct. When i changed password, I can login only on one device. Please help me.

**Comment by alidehkhodaei** [comment, marked as answer]
I changed my password once again and it worked.

**Reply by athertonwj24** [reply]
How did you change your password?

**Reply by alidehkhodaei** [reply]
I changed the password on the account I was logged into.

**Reply by syuhadaimam49-pixel** [reply]
> How did you change your password?

Neni1383@@

**Reply by alidehkhodaei** [reply]
I think the settings

**Reply by heliosempire82** [reply]
# 1. FIND THE SERVICE MANUALLY
echo ">> SCANNING FOR WATCHER NODE..."
gcloud run services list --platform managed --region us-central1

# 2. HARD-SET THE URL (Copy the URL you see from the list above)
# If the list is empty, the deployment failed. If you see a URL, copy it.
# Example: export WATCHER_URL="https://helios-watcher-node-xyz.a.run.app"

# AUTOMATIC ATTEMPT TO GRAB URL:
export WATCHER_URL=$(gcloud run services describe helios-watcher-node --platform managed --region us-central1 --format 'value(status.url)')

echo ">> TARGET ACQUIRED: $WATCHER_URL"

# 3. FIRE TEST AGAIN (Verbose Mode - Shows Errors)
curl -v -X POST "$WATCHER_URL/watcher/scan" \
     -H "Content-Type: application/json" \
     -d '{"identity": "heliosempire82@gmail.com"}'

**Comment by syuhadaimam49-pixel** [comment]
> ### Select Topic Area
> Question
> 
> ### Body
> Hello, I can login only one device and when i want login another device, it say "incorrect username or password" but username and password are correct. When i changed password, I can login only on one device. Please help me.

**Reply by alidehkhodaei** [reply]
I changed the password on the device that was already logged in.

**Comment by athertonwj24** [comment]
Hi,Again no vontect. What is this?BillSent from my Verizon, Samsung Galaxy smartphone
-------- Original message --------From: Ali Dehkhodaei ***@***.***> Date: 11/21/25  12:28 AM  (GMT-05:00) To: community/community ***@***.***> Cc: athertonwj24 ***@***.***>, Comment ***@***.***> Subject: Re: [community/community] I want login my account on other device
  (Discussion #45130) 
I changed the password on the device that was already logged in.

**Comment by athertonwj24** [comment]
Hi,What is this relating to? Thetr is no email ftpm me that msy havr initiayed this and just 1 line with no contexr.Explain please.BillSent from my Verizon, Samsung Galaxy smartphone
-------- Original message --------From: Ali Dehkhodaei ***@***.***> Date: 11/21/25  12:23 AM  (GMT-05:00) To: community/community ***@***.***> Cc: athertonwj24 ***@***.***>, Comment ***@***.***> Subject: Re: [community/community] I want login my account on other device
  (Discussion #45130) 
I think the settings

**Reply by alidehkhodaei** [reply]
Please write your message correctly; I didn’t understand it at all.

**Comment by athertonwj24** [comment]
Well, since you didn't include my reply to your original absolutely nonsensical message, i figute we're even. Thanks for nothing..Sent from my Verizon, Samsung Galaxy smartphone
-------- Original message --------From: Ali Dehkhodaei ***@***.***> Date: 11/21/25  12:30 PM  (GMT-05:00) To: community/community ***@***.***> Cc: athertonwj24 ***@***.***>, Comment ***@***.***> Subject: Re: [community/community] I want login my account on other device
  (Discussion #45130) 
Please write your message correctly; I didn’t understand it at all.
